### PR TITLE
html validation at catalog

### PR DESCRIPTION
**Pull request by ArunTeltia**
Working on #2798

This pr is the continuation of https://github.com/internetarchive/openlibrary/pull/4284

### Stakeholders

@mekarpeles @jdlrobson

### CODE REVIEW

File: openlibrary/catalog/edition_merge/templates/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h1>Merge duplicate editions</h1>
 
 {{ pager() }}
 
-{{ total }} items need to be merged ({{ easy }} easy)<p>
+{{ total }} items need to be merged ({{ easy }} easy)</p>
```
Comment (jdlrobson) [inline openlibrary/catalog/edition_merge/templates/index.html:35]: this doesn't look right to me. I think the problem here is that the macro pager creates a p tag but doesn't xlose it. Why don't we change the pager method to accept a string

e.g. `{{ pager( '{{ %s }} items need to be merged ({{ %s }} easy)'%(total, easy) }}`

OR let's not open that p tag in the macro pager.

**Review by jdlrobson — CHANGES_REQUESTED** [review]

File: openlibrary/catalog/edition_merge/templates/index.html (hunk 1 above)
Comment (ArunTeltia) [reply openlibrary/catalog/edition_merge/templates/index.html:35]: okay working on it

**Comment by jdlrobson** [comment]
Is there anything I can do to help with this one @ArunTeltia  ?

**Comment by ArunTeltia** [comment]
Can you have a look now

File: openlibrary/catalog/edition_merge/templates/index.html [hunk 2]
```diff
@@ -27,12 +27,12 @@ <h1>Merge duplicate editions</h1>
...
 {% endif %}
 {% endfor %}
-<p>
+
 {% endmacro %}
 
 {{ pager() }}
 
-{{ total }} items need to be merged ({{ easy }} easy)<p>
+{{ total }} items need to be merged ({{ easy }} easy)
```
Comment (jdlrobson) [inline openlibrary/catalog/edition_merge/templates/index.html:35]: should this entire line be inside a p? i'm wondering if the <p> at the end was meant to close the one inside the macro originally?

e.g. 
`<p> {{ total }} items need to be merged ({{ easy }} easy)</p>`

File: openlibrary/catalog/edition_merge/templates/index.html [hunk 3]
```diff
@@ -27,12 +27,12 @@ <h1>Merge duplicate editions</h1>
 <a href="?page={{ i + 1 }}">{{ i + 1}}</a>
 {% endif %}
 {% endfor %}
-<p>
+
```
Comment (jdlrobson) [inline openlibrary/catalog/edition_merge/templates/index.html:30]: 👍

**Review by jdlrobson — APPROVED** [review]

**Comment by jdlrobson** [comment]
What URI are you testing these changes on @arunteltia ?

**Comment by ArunTeltia** [comment]
Actually I was not checking in any URI I had just created a manual list in a paper and I am just checking HTML validation in each page manually.

**Comment by jdlrobson** [comment]
> Actually I was not checking in any URI I had just created a manual list in a paper and I am just checking HTML validation in each page manually.

Please do not do this in the future as this is a very important part of the code review process and can save everyone a lot of time. If you are not sure about things questions are encouraged. Particularly with any frontend change you'll want to know which page to test on before taking on a patch.

Having looked at this myself, I'm now convinced this is dead code, so we've wasted a few hours on code that is not running anywhere.

@cdrini  @mekarpeles could you please clarify how merge feature works? From looking at the code, I'm pretty sure the merge feature doesn't use any templates and if that's true we should just delete them all.

**Comment by cdrini** [comment]
Unfortunately this does look like dead code :( Sorry for the run-around @ArunTeltia ! And sorry to you both that you had to spend time fixing dead code 😓 Looks like we can delete `editions_merge` directory entirely and anything that references it :+1:

**Comment by cdrini** [comment]
In general, any .html with `{{ ... }}` style interpolation instead of `$( ... )` is very likely dead code.

**Comment by cdrini** [comment]
Created issue here: #4722

**Comment by ArunTeltia** [comment]
No problem sir working on it ☺️

**Pull request closed without merging**